frontend/globals/unity-util.ts

```diff
@@ -1431,6 +1453,8 @@ export class UnityUtil {
 		up: [number],
 		forward: [number],
 		lookAt: [number],
+		isOrthographic?: boolean,
```

Comment:
viewpoint should send in `type`? unity should be outputing the exact schema with `type` so it should take in `type` as well.

Reply:
@carmenfan At the moment, Message Parser in Unity doesn't know about the strings `orthogonal` or `perspective`. The property `orthographic` for the camera is also a `boolean`.